packages/block-editor/src/components/use-on-block-drop/index.js

```diff
@@ -144,13 +144,13 @@ export function onBlockDrop(
 export function onFilesDrop(
 	targetRootClientId,
 	targetBlockIndex,
```

Comment:
These functions should probably be rewritten a bit. There's some unused variables like this one, and I think other opportunities to use selector functions in callbacks instead of store subscriptions.